project/iir.json

```diff
@@ -10,5 +10,5 @@
     "vendors": {
         "quicklogic": ["chandalar", "quickfeather"]
     },
-    "required_toolchains": ["quicklogic"]
```

Comment:
This should be a seperate pull request?

Reply:
I have added a strict check to have no missing frequencies in the meta.json. This test is not able to produce frequency outputs, as there isn't a real clock net driving logic (this is a dummy test, not meant to run on HW as far as I have learned).